src/systems/spawning/spawn_player_base.rs

```diff
@@ -3,3 +3,49 @@
 // E.G 2 Players spawn A could be bottom leftmost tile with spawn B the top top rightmost
 // E.G 4 Players spawn A, B, C, D in each corner
 // Minimum map size should be player count
+
+use bevy::{
+    audio::CpalSample,
+    ecs::{
+        event::EventWriter,
+        system::{Commands, Res},
+    },
+    math::Quat,
+    transform::components::Transform,
+};
+use rand::random;
+
+use crate::{
+    components::spacestation::SpaceStation,
+    events::{spawn_planet_event::SpawnPlanetEvent, spawn_sprite_event::SpawnSpriteEvent},
+    resources::{constants::SPACESTATION_DISTANCE_FROM_CENTRE, game_settings::GameSettings},
+};
+
+pub fn spawn_space_stations(
+    game_settings: Res<GameSettings>,
+    mut commands: Commands,
+    mut spawn_sprite_event: EventWriter<SpawnSpriteEvent>,
+    mut spawn_planet_event: EventWriter<SpawnPlanetEvent>,
+) {
+    let angle = 360.0 / game_settings.number_of_players.to_float_sample();
+
+    for player in 1..game_settings.number_of_players + 1 {
```

Comment:
@MatthewGriffin 0..game_settings.number_of_players?

Reply:
(Addressed)

Reply:
I did 1..noPlayers +1 as I thought that / 0 in the angle calc would cause problems but since it seems to be ok with that then its fine as 0..players